packages/augur-ui/src/modules/modal/cash-out-form.tsx

```diff
@@ -28,34 +37,65 @@ interface CashOutFormProps {
   reserveInDaiFormatted: FormattedNumber;
   totalDaiFormatted: FormattedNumber;
   tradingAccountEthFormatted: FormattedNumber;
+  totalDai: string;
 }
 
-export const CashOutForm = ( props: CashOutFormProps) => {
+export const CashOutForm = (props: CashOutFormProps) => {
```

Comment:
Since you touched that file, I would suggest removing the destructuring from the first line of this function and just adding to the parameters, like so:

```jsx
export const CashOutForm = ({
    closeAction,
    withdrawAllFunds,
    withdrawAllFundsEstimateGas,
    account,
    gasPrice,
    totalOpenOrderFundsFormatted,
    availableFundsFormatted,
    reserveInDaiFormatted,
    totalDaiFormatted,
    tradingAccountEthFormatted,
  }: CashOutFormProps) => {
```

Reply:
keeping the format. following how betting ui is doing it. it'll make merging easier for them

Reply:
good idea. I'll do that